### PR TITLE
Implement FakeViewModel and FakeViewModelTest for "Contributor" screen.

### LINKED ISSUE #21: Implement FakeViewModel and FakeViewModelTest for "Contributor" screen
## Overview (Required)
I want to make app "Testable & Previewable & Faster debug" as in https://github.com/DroidKaigi/conference-news-app-2021#testable--previewable--faster-debug .
That's why I want Fake to behave as much as possible with a real ViewModel. So add FakeViewModel and the corresponding tests.
## Links
https://github.com/DroidKaigi/conference-news-app-2021/blob/main/uicomponent-compose/news/src/main/java/io/github/droidkaigi/confnews2021/staff/news/FakeNewsViewModel.kt

### PR DESCRIPTION
**Pull request by tick-taku**
## Issue

- close #21 

## Overview (Required)

- Create 
  - Contributor data class.
  - ContributorViewModel interface.
  - FakeContributorViewModel.
  - ContributorViewModelTest.

## Review point

- [x] Each class's package is ok?
- [x] Contributor data class's properties is ok?
- [x] Are the test case enough?
- [x] RealViewModel is not included this PR.

### CODE REVIEW

**Comment by tick-taku** [comment]
I'm sorry to keep you waiting.💦  It was fun to make new discoveries!!
Please review!

**Comment by takahirom** [comment]
👀

File: android/src/test/java/io/github/droidkaigi/feeder/ContributorViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+@InternalCoroutinesApi
+@RunWith(Parameterized::class)
+class ContributorViewModelTest(
```
Comment (takahirom) [inline android/src/test/java/io/github/droidkaigi/feeder/ContributorViewModelTest.kt:17]: Thank you for creating test👍

**Review by takahirom — APPROVED** [review]
LGTM 👍 Thank you for creating the first step of contributor screen!!!

**Comment by takahirom** [comment]
Sorry I didn't see the review point. I will check this 👀

**Comment by takahirom** [comment]
Thank you! I checked it! ✅

**Pull request merged**